Add some discussion about future directions (#58)
adamginsburg/astroquery-paper: main.tex

JPLHorizons module has been implemented as part of the \texttt{sbpy}
project\footnote{\url{http://sbpy.org}} with support from NASA PDART grant
80NSSC18K0987. Further Solar System-related services are planned to be added to
\astroquery through this support. \astroquery has also received support from the Google Summer of Code
program, with two students (co-authors Madhura Parikh and Simon Liedtke)
from 2013--2014.

> Due to its nature as a open developed ``big tent'' package, \astroquery intentionally

Reviewer: I don't like "big tent"; I had never heard the colloquialism and had to look it up
```suggestion
Due to its nature as an openly developed package, \astroquery intentionally
```

Reviewer 2: agreed

does not have a long-term roadmap for future features. New directions are primarily 
driven by contributors and data providers adding or updating modules to reflect new 
or changed data  sources. The underlying software architecture has been demonstrably
sufficient to meet the needs of the  current generation of data sources (proven by  
the user base of \astroquery).
While this may change in the future, the user-focused nature of \astroquery means

> that making such changes is unnecessary until there are specific data sources 

Reviewer: ```suggestion
that making such changes is unnecessary until there are specific data sources or use cases
```

Author: WHOA! This "suggested change" thing is blowing my mind!
